app/upgrades.go

```diff
@@ -136,6 +141,32 @@ func ConvertEOAsToBaseAccount(ctx sdk.Context, accountKeeper authkeeper.AccountK
 	})
 }
 
+func AddKavadistFundAccount(ctx sdk.Context, accountKeeper authkeeper.AccountKeeper, bankKeeper bankKeeper.Keeper, distKeeper distrkeeper.Keeper) {
+	maccAddr := accountKeeper.GetModuleAddress(kavadisttypes.FundModuleAccount)
+
+	accountI := accountKeeper.GetAccount(ctx, maccAddr)
+	// if account already exists and is a module account, return
```

Comment:
assuming only we/the chain can create new module accounts (e.g. not something that can be done over the api or cli)?

Reply:
Correct, usually the are instatiated at `initGenesis`, but since this module already exists, we need to add it manually in the migration.